Code/GraphMol/ChemTransforms/testChemTransforms.cpp

```diff
@@ -173,6 +194,30 @@ void testReplaceSubstructs() {
   TEST_ASSERT(vect[0]->getNumAtoms() == 3);
   TEST_ASSERT(vect[0]->getNumBonds() == 3);
 
+  // test chirality
```

Comment:
Same as above: both examples need the test case that shows that the replacement happens when useChirality is false.